examples/simple-pipeline/assets/hello_ingestr7_asset.yml

```diff
@@ -0,0 +1,9 @@
+name: public.stripe
+type: ingestr
+connection: postgres
+
+parameters:
+  source_connection: new_stripe
```

Comment:
maybe for the sake of not making it confusing, let's give the source connection the same name as [here](https://github.com/bruin-data/bruin/pull/87/files#diff-d3df4c32515d23a56e039eb1897c46b80d2ddcab4eca491282fcab99b47695e6R21)

Reply:
I made changes and pushed it.